faassupervisor/storage/providers/dCache.py

```diff
@@ -0,0 +1,30 @@
+from faassupervisor.storage.providers import DefaultStorageProvider
+from webdav3.client import Client
+
+class DCache(DefaultStorageProvider):
```

Comment:
@gmolto can we generalize this and name the storage provider "webdav"?

Reply:
Yes, I agree. Excellent idea.